docs/guides/esp32/ota.md

```diff
@@ -35,11 +35,24 @@ control list.
 
 ### Using Console
 
-After commissioning is successful, press Enter in requestor device console and
-type below query.
+After commissioning is successful, read the default-ota-providers list of
+requestor.
 
 ```
->matter ota query 1 <PROVIDER NODE ID> 0
+./out/debug/chip-tool otasoftwareupdaterequestor read default-otaproviders <REQUESTOR NODE ID> 0
+```
+
+If list do not have your provider, write into default-ota-providers list of
+requestor.
+
+```
+./out/debug/chip-tool otasoftwareupdaterequestor write default-otaproviders '[{"fabricIndex": 1, "providerNodeID": <PROVIDER_NODE_ID>, "endpoint": 0}, {"fabricIndex": 1, "providerNodeID": <PROVIDER_NODE_ID>, "endpoint": 0}]' <REQUESTOR_NODE_ID> 0
```

Comment:
Why is the same thing repeated twice?

Reply:
So this is confusing. You can mention provider1 node id, provider2 node id

Reply:
Done